Lib/random.py

```diff
@@ -84,13 +86,26 @@ class Random(_random.Random):
 
     VERSION = 3     # used by getstate/setstate
 
-    def __init__(self, x=None):
+    def __init__(self, seed=None, x=_sentinel):
```

Comment:
Hi @Carreau, is there a reason not to use `None` as the sentinel value here ?

Reply:
Detect people that would already call `Random(x=None)` directly or indirectly.
That should also emit a warning if this get deprecated, does that make sens ?

Reply:
Indeed, thanks very much for the precision